pkg/workceptor/command.go

```diff
@@ -111,6 +112,13 @@ loop:
 			err = status.UpdateBasicStatus(statusFilename, WorkStateRunning, fmt.Sprintf("Running: PID %d", cmd.Process.Pid), stdoutSize(unitdir))
 			if err != nil {
 				logger.Error("Error updating status file %s: %s", statusFilename, err)
+				writeStatusFailures++
+				if writeStatusFailures > 3 {
+					logger.Error("Exceeded retries for updating status file %s: %s", statusFilename, err)
+					os.Exit(-1)
```

Comment:
https://pkg.go.dev/os#Exit: "For portability, the status code should be in the range [0, 125].". Also os.Exit causes the process to exit immediately without executing any defers and such. Is that desired?